src/main/java/DeadLine.java

```diff
@@ -0,0 +1,16 @@
+public class DeadLine extends Task
+{
+    private String deadLineBy;
```

Comment:
Perhaps you could check if it is spelled deadline or dead line. The uppercase  "L"  affects code readability as It might be interpreted as "dead line" which might  be interpreted differently from "deadline" 

```suggestion
    private String deadlineBy;
```